cli/azd/main.go

```diff
@@ -6,357 +6,14 @@
 package main
 
 import (
```

Comment:
While moving the logic out, we also need to reorder when the cobra.Command gets constructed. You can view the diff in this [commit](https://github.com/Azure/azure-dev/pull/2345/commits/5fc288a05e5ba510f4ea0fa8092bfff34084137e).